Traceback (most recent call last):
  File "/Users/rajarshi/miniforge/envs/web/lib/python3.12/site-packages/jupyter_cache/executors/utils.py", line 58, in single_nb_execution
    executenb(
  File "/Users/rajarshi/miniforge/envs/web/lib/python3.12/site-packages/nbclient/client.py", line 1305, in execute
    return NotebookClient(nb=nb, resources=resources, km=km, **kwargs).execute()
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/rajarshi/miniforge/envs/web/lib/python3.12/site-packages/jupyter_core/utils/__init__.py", line 173, in wrapped
    return loop.run_until_complete(inner)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/rajarshi/miniforge/envs/web/lib/python3.12/asyncio/base_events.py", line 664, in run_until_complete
    return future.result()
           ^^^^^^^^^^^^^^^
  File "/Users/rajarshi/miniforge/envs/web/lib/python3.12/site-packages/nbclient/client.py", line 705, in async_execute
    await self.async_execute_cell(
  File "/Users/rajarshi/miniforge/envs/web/lib/python3.12/site-packages/nbclient/client.py", line 1058, in async_execute_cell
    await self._check_raise_for_error(cell, cell_index, exec_reply)
  File "/Users/rajarshi/miniforge/envs/web/lib/python3.12/site-packages/nbclient/client.py", line 914, in _check_raise_for_error
    raise CellExecutionError.from_cell_and_msg(cell, exec_reply_content)
nbclient.exceptions.CellExecutionError: An error occurred while executing the following cell:
------------------
decimal = int(input('Enter an integer in decimal: '))

old_binary = bin(decimal)
print('Original binary = ',old_binary)
old_binary = old_binary[2:]

if len(old_binary) <= 10:
    zeros = '0'*(10 - len(old_binary))
    new_binary = zeros + old_binary
else:
    new_binary = old_binary[-10:]

print('10 bit binary = ',new_binary)
------------------


[0;31m---------------------------------------------------------------------------[0m
[0;31mStdinNotImplementedError[0m                  Traceback (most recent call last)
Cell [0;32mIn[2], line 1[0m
[0;32m----> 1[0m decimal [38;5;241m=[39m [38;5;28mint[39m([38;5;28;43minput[39;49m[43m([49m[38;5;124;43m'[39;49m[38;5;124;43mEnter an integer in decimal: [39;49m[38;5;124;43m'[39;49m[43m)[49m)
[1;32m      3[0m old_binary [38;5;241m=[39m [38;5;28mbin[39m(decimal)
[1;32m      4[0m [38;5;28mprint[39m([38;5;124m'[39m[38;5;124mOriginal binary = [39m[38;5;124m'[39m,old_binary)

File [0;32m~/miniforge/envs/web/lib/python3.12/site-packages/ipykernel/kernelbase.py:1250[0m, in [0;36mKernel.raw_input[0;34m(self, prompt)[0m
[1;32m   1248[0m [38;5;28;01mif[39;00m [38;5;129;01mnot[39;00m [38;5;28mself[39m[38;5;241m.[39m_allow_stdin:
[1;32m   1249[0m     msg [38;5;241m=[39m [38;5;124m"[39m[38;5;124mraw_input was called, but this frontend does not support input requests.[39m[38;5;124m"[39m
[0;32m-> 1250[0m     [38;5;28;01mraise[39;00m StdinNotImplementedError(msg)
[1;32m   1251[0m [38;5;28;01mreturn[39;00m [38;5;28mself[39m[38;5;241m.[39m_input_request(
[1;32m   1252[0m     [38;5;28mstr[39m(prompt),
[1;32m   1253[0m     [38;5;28mself[39m[38;5;241m.[39m_parent_ident[[38;5;124m"[39m[38;5;124mshell[39m[38;5;124m"[39m],
[1;32m   1254[0m     [38;5;28mself[39m[38;5;241m.[39mget_parent([38;5;124m"[39m[38;5;124mshell[39m[38;5;124m"[39m),
[1;32m   1255[0m     password[38;5;241m=[39m[38;5;28;01mFalse[39;00m,
[1;32m   1256[0m )

[0;31mStdinNotImplementedError[0m: raw_input was called, but this frontend does not support input requests.

